src/main/java/app/linguistai/bmvp/repository/wordbank/IUnknownWordRepository.java

```diff
@@ -12,4 +13,7 @@
 public interface IUnknownWordRepository extends JpaRepository<UnknownWord, UnknownWordId> {
     List<UnknownWord> findByOwnerListListId(UUID listId);
     Optional<UnknownWord> findByOwnerListListIdAndWord(UUID listId, String word);
+
+    @Query("SELECT u.confidence, COUNT(u) FROM UnknownWord u WHERE u.ownerList.user.id = :userId GROUP BY u.confidence")
+    List<Object[]> countWordsByConfidenceLevel(@Param("userId") UUID userId);
```

Comment:
Is it possible for the list type to be `UnknownWord` instead of `Object[]`? Using `Object[]` may not be the best practice and may introduce some bugs later on

Reply:
Changed it to return an interface, following this guide (under native queries):
https://stackoverflow.com/a/36329166